test/tests/pbs_smoketest.py

```diff
@@ -329,18 +329,29 @@ def test_limits(self):
     @skipOnCpuSet
     def test_finished_jobs(self):
         """
-        Test for finished jobs
+        Test for finished jobs and resource used for jobs.
         """
         a = {'resources_available.ncpus': '4'}
         self.server.manager(MGR_CMD_SET, NODE, a, self.mom.shortname)
         a = {'job_history_enable': 'True'}
         self.server.manager(MGR_CMD_SET, SERVER, a)
-        a = {'Resource_List.walltime': '10', ATTR_k: 'oe'}
-        j = Job(TEST_USER, attrs=a)
-        j.set_sleep_time(5)
+        a = {'Resource_List.select': '1:ncpus=2:mem=20mb',
+             'Resource_List.walltime': '15'}
```

Comment:
we can remove walltime from select as we are only checking for it to be non-zero.

Reply:
In fact, we can have select to only be "ncpus=2"